student.rb

```diff
@@ -3,9 +3,9 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(id, age, classroom, parent_permission: true, name: 'Unknown')
+  def initialize(id, age, _classroom, parent_permission: true, name: 'Unknown')
     super(id, age, parent_permission: parent_permission, name: name)
-    @classroom = classroom
+    @classroom = nil
   end
 
```

Comment:
- [ ] Kindly make sure to create a method that serves as the 'belongs-to' side (a student belongs to a classroom) this method should accept a classroom argument and assign the classroom attribute as well as push the student to the given classroom. [Refer to this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for further guidance